examples/resources/awscc_rds_db_cluster/rds_db_cluster_create.tf

```diff
@@ -0,0 +1,7 @@
+resource "awscc_rds_db_cluster" "example_db_cluster" {
+  availability_zones    = ["us-east-1b", "us-east-1c"]
+  engine                = "aurora-mysql"
+  db_cluster_identifier = "example-dbcluster"
+  master_username       = "foo"
+  master_user_password  = "foobarbaz"
+}
```

Comment:
the file name should call it as rds_db_cluster.tf

Reply:
completed.